packages/story-editor/src/components/form/useHotlink.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { useCallback, useMemo, useState } from '@googleforcreators/react';
+import { __, sprintf } from '@googleforcreators/i18n';
+import { getExtensionsFromMimeType } from '@googleforcreators/media';
+import { v4 as uuidv4 } from 'uuid';
+/**
+ * Internal dependencies
+ */
+import getResourceFromUrl from '../../app/media/utils/getResourceFromUrl';
+
+function useHotlink({ onChange, cropParams, type }) {
```

Comment:
`cropParams` is confusing since we don't actually "crop" any images. It's about requiring certain dimensions.

Let's try to find a better name here.